sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -838,6 +850,11 @@ def get_laser_temperature(self):
                            'high warn': laser_temp_high_warn,
                            'low warn': laser_temp_low_warn}
         return laser_temp_dict
+    
+    def _get_laser_temp_threshold(self, field):
```

Comment:
what about other fields which can result in read failure

Reply:
I have added a UT case to randomly return None for EEPROM read. For now, I fixed all issues found by the UT case. For other field which I did not touch, I assume the return value is properly checked/handled (not 100% sure).